README.md

```diff
@@ -68,6 +68,19 @@ func open(sender: AnyObject) {
 }
 ```
 
+You can also use your own FolioReader view controller like this.
+
+```swift
+		let config = FolioReaderConfig()
+
+		let bookPath = NSBundle.mainBundle().pathForResource("book", ofType: "epub")
+		let epubVC = FolioReaderContainer(config: config, epubPath: bookPath, removeEpub: true)
```

Comment:
Have you tested this? This init is not public, we need to make it public to work.